nixos/modules/services/x11/hardware/cmt.nix

```diff
@@ -30,6 +30,7 @@ in {
 
   config = mkIf cfg.enable {
 
+    services.xserver.libinput.enable = false; #plasma5.nix enables libinput by default which is incompatible with cmt.nix
```

Comment:
We have this assertion below
`!config.services.xserver.libinput.enable`
which produces a message
```
cmt and libinput are incompatible, you cannot enable both (in services.xserver).
```

I think the actual issue is the module doesn't prompt the user to do any proactive change.
Can we change the message so it tells the users to disable `services.xserver.libinput.enable`?

Reply:
Sure that could also work.